tests/integration/pipeline.ts

```diff
@@ -403,6 +403,11 @@ const createRuntimeConfig = (): any => {
   cy.get('.elyra-metadataEditor-form-cos_username').type('minioadmin');
   cy.get('.elyra-metadataEditor-form-cos_password').type('minioadmin');
   cy.get('.elyra-metadataEditor-form-cos_bucket').type('test-bucket');
+  // Check the default value is displayed on github api endpoint field
+  cy.get('input.elyra-metadataEditor-form-github_api_endpoint').should(
```

Comment:
is the test failing due to the extra `input.` on the `get`?

Reply:
Yes, apparently it can't find it, I'll take a look. Couldn't run cypress on my machine yesterday, so I was waiting on the CI results here.

Reply:
This assertion is now passing. 
The problem now is saving a new runtime is triggering validation error when it shouldn't
![image](https://user-images.githubusercontent.com/25207344/109326157-cee1ea80-7824-11eb-9011-d5a5622aca28.png)